mapbox_tilesets/scripts/cli.py

```diff
@@ -197,6 +197,33 @@ def update(
         raise errors.TilesetsError(r.text)
 
 
+@cli.command("delete")
+@click.argument("tileset", required=True, type=str)
+@click.option("--force", "-f", is_flag=True, help="Circumvents confirmation prompt")
+@click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
+@click.option("--indent", type=int, default=None, help="Indent for JSON output")
+def delete(tileset, token=None, indent=None, force=None):
+    """Delete your tileset.
+
+    tilesets delete <tileset_id>
+    """
+
+    mapbox_api = _get_api()
+    mapbox_token = _get_token(token)
+
+    if not force:
+        click.confirm(
+            "Are you sure you want to delete {0}?".format(tileset), abort=True
```

Comment:
Thoughts on prompting users to type the tileset name to add a little more friction?

Reply:
Also, adding some warning text to the docs, e.g. "this action is not recoverable"

Reply:
I can, should I do the same for the `delete-source` command?

Reply:
Sure, makes sense to me!